hasktorch/src/Torch/Functional.hs

```diff
@@ -838,26 +868,35 @@ maxPool2d kernelSize stride padding dilation ceilMode self =
         where
             asList :: (Int, Int) -> [Int]
             asList (a0, a1) = [a0, a1]
- 
--- | Applies a 3D max pooling over an input signal composed of several input planes.
-maxPool3d 
-    :: (Int,Int,Int) -- ^ kernel size
-    -> (Int,Int,Int) -- ^ stride
-    -> (Int,Int,Int) -- ^ padding
-    -> (Int,Int,Int) -- ^ dilation
-    -> Bool -- ^ ceil mode
-    -> Tensor -- ^ input
-    -> Tensor -- ^ output
-maxPool3d kernelSize stride padding dilation ceilMode self =
-    unsafePerformIO $ (cast6 ATen.max_pool3d_tllllb)
-        self kernelSize stride padding dilation ceilMode
```

Comment:
Should we remove maxPool3d?

Reply:
I think this was accidental as I was moving the maxpool ops to another spot in the file. Thanks for the catch.